vyper/ir/compile_ir.py

```diff
@@ -23,7 +23,8 @@ def num_to_bytearray(x):
 
 def PUSH(x):
```

Comment:
wonder if this might be better named `push_stack` or something clearer, I thought it was an opcode at first

Reply:
could be -- it is supposed to be an abstraction of the PUSH opcodes